src/custom-elements/index.js

```diff
@@ -1,9 +1,18 @@
 import '@webcomponents/custom-elements';
 import { defineCustomElement } from 'vue';
-import indicator from './Indicator.vue';
+import Indicator from './Indicator.vue';
 import { initEventListenersAndStore } from './webapp-events';
 
 initEventListenersAndStore();
 
-let indicatorCustomElement = defineCustomElement(indicator);
-customElements.define('overlay-indicator', indicatorCustomElement);
+// Workaround for https://github.com/vuejs/core/issues/4662
+const modules = import.meta.glob('./**/*.vue');
+Promise.all(Object.values(modules).map((module) => module())).then((modules) => {
+  const styles = modules.map((module) => module.default.styles);
+  Indicator.styles = [styles.flat().join('')];
+
+  let indicatorCustomElement = defineCustomElement(Indicator);
+  customElements.define('overlay-indicator', indicatorCustomElement);
+
+  console.log('Custom element defined');
+});
```

Comment:
Check the bundle change

Reply:
I haven't seen any difference (except a lot of diffs of `Xi -> Zi`). I guess it is because the diff will be just a string of styles that becomes longer.